### PR TITLE
GitHub Release Assets

### PR DESCRIPTION
**Pull request by scopatz**
CC @CJ-Wright @justcalamari

### CODE REVIEW

File: rever/activities/ghrelease.xsh [hunk 1]
```diff
@@ -48,6 +49,13 @@ class GHRelease(Activity):
         defaults to ''
     :$GHRELEASE_APPEND: str, string to append to the release notes,
         defaults to ''
+    :$GHRELEASE_ASSETS: list of str or functions, Extra assests to
```
Comment (CJ-Wright) [inline rever/activities/ghrelease.xsh]: Pedantic hat on: shouldn't this be an iterable of str or functions? (since we could pass a tuple)

**Comment by CJ-Wright** [comment]
Should we offer the option to remove the asset after it is uploaded? It might be odd to be carrying around a generated asset in the source.

**Comment by scopatz** [comment]
Ok, I have added a git archive default, which does solve #130. 

> Should we offer the option to remove the asset after it is uploaded? It might be odd to be carrying around a generated asset in the source.

The best practice is probably for these assets to be created in an ignored place, such as `$REVER_DIR`, or for python, the `dist/` dir

**Comment by CJ-Wright** [comment]
LGTM

**Pull request merged**